src/Wallabag/CoreBundle/Repository/TagRepository.php

```diff
@@ -51,4 +51,20 @@ public function findAllTags($userId)
             ->getQuery()
             ->getResult();
     }
+
+    /**
+     * Used only in test case to get a tag for our entry.
+     *
+     * @return Tag
+     */
+    public function findOnebyEntryAndLabel($entry, $label)
```

Comment:
`By` instead of `by`. Also what do you think about `findOneByEntryAndTagLabel` instead?